_data/not_edited_here.yaml

```diff
@@ -10,29 +10,30 @@
 
 # Make sure directories have the trailing slash, keep the list alphabetical
 
-- path: apidocs/
+overrides:
+- path: /apidocs/
   description: "Auto-generated API docs for Docker Cloud, DTR, UCP. File an issue."
 
-- path: engine/deprecated.md
+- path: /engine/deprecated.md
   description: Docker Engine deprecation reference
-  source: https://github.com/docker/docker/docs/deprecated.md
+  source: https://github.com/docker/docker/tree/master/docs/deprecated.md
```

Comment:
One potential fly in the ointment. If they make the change in `docker/docker` `master`, it will need to be cherry-picked backward to `docker/docker` <release_branch> (1.12.x currently, soon to be 1.13.x I guess) in order to be published. I don't really know how we could help with this. They will also need to figure out how to sign the commit because of `docker/docker` requirements. I guess once they leave our repo, they are on their own, but I can see this being a point of friction.